assets/src/scss/layout/navbar/_burger-menu.scss

```diff
@@ -13,7 +13,7 @@
   left: 0;
   inset-inline-start: -100%;
   padding: 18px $sp-3;
-  z-index: 99999;
+  z-index: 9999999999;
```

Comment:
This is added because the Feedback button, from the NL site, is overlapping the burger menu.